### PR TITLE
hotfix/move_token_to_local-storage

### PR DESCRIPTION
**Pull request by K0sen**

### CODE REVIEW

File: frontend/src/services/common/storageService.js [hunk 1]
```diff
@@ -12,6 +13,16 @@ export const storageService = {
     setToken(token) {
         return this.set(this.keyName, token);
     },
+    hasToken() {
+        console.log(this.get(this.keyName));
```
Comment (nemoipaha) [inline frontend/src/services/common/storageService.js]: Remove)

File: frontend/src/services/common/storageService.js [hunk 2]
```diff
@@ -12,6 +13,16 @@ export const storageService = {
...
+        return !!this.get(this.keyName);
+    },
+    setAuthenticatedUser(user) {
+        this.set(this.USER, JSON.stringify({email: user.email}));
```
Comment (nemoipaha) [inline frontend/src/services/common/storageService.js]: JSON.stringify(user) better

File: frontend/src/store/modules/auth/getters.js [hunk 3]
```diff
@@ -1,5 +1,9 @@
+import StorageService from '@/services/common/storageService';
+
 export default {
-    isLoggedIn: state => !!state.token,
-    getAuthenticatedUser: state => state.currentUser,
-    getToken: state => state.token,
-}
\ No newline at end of file
+    isLoggedIn: (state) => {
+        return StorageService.hasToken() || !!state.token;
```
Comment (nemoipaha) [inline frontend/src/store/modules/auth/getters.js]: StorageService.hasToken() should be enough

File: frontend/src/services/common/storageService.js [hunk 4]
```diff
@@ -12,6 +13,18 @@ export const storageService = {
     setToken(token) {
         return this.set(this.keyName, token);
     },
+    hasToken() {
+        return !!this.get(this.keyName);
+    },
+    setAuthenticatedUser(user) {
+        this.set(this.USER, JSON.stringify({email: user.email}));
```
Comment (nemoipaha) [inline frontend/src/services/common/storageService.js]: JSON.stringify(user) is ok

File: frontend/src/store/modules/auth/getters.js (hunk 3 above)
Comment (K0sen) [reply frontend/src/store/modules/auth/getters.js]: i have done like that but record a token to local storage does not trigger authentification. Should be another way. Investigating it

File: frontend/src/services/common/storageService.js (hunk 4 above)
Comment (K0sen) [reply frontend/src/services/common/storageService.js]: Ok

**Comment by K0sen** [comment]
fixed in other way

**Pull request merged**